README.md

```diff
@@ -64,22 +64,26 @@ APNS | ✅ | ❌ | [kylebrowning/APNSwift](https://github.com/kylebrowning/APNSw
 PostgreSQL | ✅ | ❌ | [vapor/postgres-nio](https://github.com/vapor/postgres-nio) | `PostgresNIO` | SSWG community project
 Redis | ✅ | ❌ | [mordil/swift-redi-stack](https://gitlab.com/Mordil/swift-redi-stack) | `RediStack` | SSWG community project
 
-### Supported Platforms
+### Supported Versions
 
-SwiftNIO aims to support all of the platforms where Swift is supported. Currently, it is developed and tested on macOS and Linux, and is known to support the following operating system versions:
+### SwiftNIO 2
+This is the current version of SwiftNIO and will be supported for the forseeable future.
 
-* Ubuntu 18.04+
-* macOS 10.9+, iOS 7+; (macOS 10.14+, iOS 12+, tvOS 12+ or watchOS 6+ with [swift-nio-transport-services][repo-nio-transport-services])
+The latest released SwiftNIO 2 version supports Swift 5.2+. NIO 2.29.0 and older support Swift 5.0+.
```

Comment:
Should we given an upper bound on 2.29.0 and older support? I.e. 5.0+ vs 5.0 ..< 5.5?

Reply:
Probably not given that someone has just made nio1 work with swift 5.5.  It would seem unreasonable to be more restrictive here.  We do have practical constraints though that we can only do patch releases and stick with sensible semantic versioning.

Reply:
Okay, I think we should be clear about upper bounds when we do the next major version though.